packages/twenty-front/src/modules/ui/input/components/TextInput.tsx

```diff
@@ -89,7 +89,6 @@ export const TextInput = ({
       onInputEnter?.();
 
       if (isDefined(inputRef) && 'current' in inputRef) {
-        inputRef.current?.blur();
```

Comment:
why is it here???
doesnt this cause an unnecessary onblur call?

Reply:
But if not removed, two cards are being created on Enter.
Which means when onEnter is called onBlur is triggered too. 
How do we handle this? or should we just not create a card onBlur and keep this as it is?
Which means card would be created only onEnter. :(

Reply:
Indeed